scripts/lib/python/nas_os_if_utils.py

```diff
@@ -705,20 +736,44 @@ def get_breakoutCap_currentMode_mode(if_index):
         return None
     return (get_port_breakoutCap_currentMode_mode(fp_port))
 
-def get_port_breakoutCap_currentMode_mode(fp_port):
-    fp_list = nas_os_fp_list(d={'front-panel-port':fp_port})
-    if fp_list == None or len(fp_list) == 0:
-        log_err('failed to get object of front panel port %d' % fp_port)
-        return None
-    fp_port_obj = cps_object.CPSObject(obj=fp_list[0])
+def get_pg_br_cap_list(pg_id):
+    pg_list = nas_os_pg_state_list(d={'dell-pg/port-groups-state/port-group-state/id':pg_id})
+    pg_port_obj = cps_object.CPSObject(obj=pg_list[0])
+    br_cap_list = pg_port_obj.get_attr_data('br-cap')
+    return br_cap_list
 
-    br_cap_list = fp_port_obj.get_attr_data('br-cap')
+def get_port_breakoutCap_currentMode_mode(fp_port):
+    fp_utils.init()
+    fp_details = fp.find_front_panel_port(fp_port)
+    if fp_details.is_pg_member() is True:
+        pg_name_list=fp.get_port_group_list()
+        pg_id = fp_details.get_port_group_id()	
+        pg = pg_name_list[pg_id]
+        print "Port-group profile type:", pg.get_profile_type()
```

Comment:
Same comment about prints.

Reply:
print "Port-group profile type:", pg.get_profile_type() - the port type - ethernet_sfp28  or qsfp .